messages/deploy/messages.go

```diff
@@ -17,11 +17,15 @@ var (
 	DeployOutputDomainCreate          = "Created Domain %v with ID %v\n"
 	DeployOutputDomainUpdate          = "Updated Domain %v with ID %v\n"
 	EdgeApplicationDeployPathFlag     = "Path to where your static files are stored"
+	OriginsSuccessful                 = "Created Origin for edge application\n"
 	CacheSettingsSuccessful           = "Created Cache Settings for Edge Application\n"
 	DeployInputAddress                = "Please inform an address to be used in the Origin of this application: "
 	RulesEngineSuccessful             = "Created Rules Engine for Edge Application\n"
 	DeployFlagHelp                    = "Displays more information about the deploy command"
 	DeployPropagation                 = "Your application is being deployed to all Azion Edge Locations and it might take a few minutes.\n"
 	UploadStart                       = "Uploading static files\n"
 	UploadSuccessful                  = "\nUpload completed successfully!\n"
+	BucketInUse                       = "This bucket's name is already in use, please type another one"
```

Comment:
Let's change this to:
This bucket's name is already in use, please try another one